tests/storage/test_client_ips.py

```diff
@@ -143,6 +144,61 @@ def test_insert_new_client_ip_none_device_id(self):
             ],
         )
 
+    def test_get_user_ip_and_agents_from_memory(self):
+        """Test that `get_user_ip_and_agents` retrieves unpersisted data from memory"""
+        self.reactor.advance(12345678)
```

Comment:
Does this perform any operations or just live the clock forward? (Should we do like 100 instead of such a large number?)

Reply:
I was following the pattern of the tests above. I think the number is arbitrary and it's really to ensure that the `last_seen` timestamp for a user has a distinctly recognizable value that couldn't arise by some other means.